app/models/episode_story_handler.rb

```diff
@@ -63,8 +63,8 @@ def update_attributes
     episode.published_at = sa[:published_at] ? Time.parse(sa[:published_at]) : nil
 
     %w(season episode).each do |time|
-      id = sa["#{time}_identifier"]
-      episode["#{time}_number"] = id.to_i if id && (!id.to_i.zero? || [0, '0'].include?(id))
+      id = sa["#{time}_identifier"].to_i
```

Comment:
This is a safe operation, as if there is no key on the `sa[]` or if has the key, but a nil, blank, 0, or non-numeric value, the `.to_i` will turn any of those cases into the int `0`.

Both season` and `episode` should be integers and > 0, so if the value is 0, we set it to nil, and if it is greater than 0, then we use the numeric value.